projects/iot-cloud/stack.yml

```diff
@@ -6,19 +6,44 @@ services:
     volumes:
       - graphite:/opt/graphite/storage
       - graphite_conf:/opt/graphite/conf
+    environment:
+      VIRTUAL_HOST: 'graphite.beia.cloud,graphite.beia-telemetrie.ro,graphite.beia-consult.ro'
+      VIRTUAL_PORT: 80
     networks:
      - graphite
+     - proxy_net
 
   grafana:
     image: grafana/grafana
     environment:
-      GF_SECURITY_ADMIN_PASSWORD: '${GRAFANA_PASSWORD:-secret}'
+      GF_SECURITY_ADMIN_PASSWORD__FILE: '/run/secrets/grafana_password'
       VIRTUAL_HOST: 'grafana.beia.cloud,grafana.beia-telemetrie.ro,grafana.beia-consult.ro'
     networks:
      - graphite
      - proxy_net
     volumes:
       - grafana:/var/lib/grafana
+    secrets:
+      - grafana_password
+
+  statsd:
+    image: vladwing/statsd
+    environment:
+      GRAPHITE_HOST: 'graphite'
+    networks:
+     - graphite
+     - statsd_net
+
+  bigdata:
```

Comment:
Let's not call it like so. I would prefer a self-explaining name link postgres.

Even though we might change postgres with some other solution as bigdata solution, it won't be directly replaceable in the code, so I see no advantage in having such a name in the stack.

Reply:
"Postgres" is a bad name as it explains only what it is, not what it does. "Data storage" sounds equally dumb, but at least "bigdata" can be seen as a joke.